pkg/manifest/jsonmanifest/entry.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package jsonmanifest
+
+import (
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/manifest"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+// verify JSONEntry implements manifest.Entry
+var _ manifest.Entry = (*JSONEntry)(nil)
```

Comment:
I know that this is out of scope for this PR, but i have a naming suggestion to drop `JSON` prefixes from types in this package. I think that json is clear from the package name. I would go even further and rename package to just `json`, but that requires package naming in imports, which I do not mind, but it may be too much. :)

Reply:
> I know that this is out of scope for this PR, but i have a naming suggestion to drop `JSON` prefixes from types in this package. I think that json is clear from the package name. I would go even further and rename package to just `json`, but that requires package naming in imports, which I do not mind, but it may be too much. :)

we've previously spoken about this with @zbiljic and we 100% agree.

we decided to keep this as-is while it is inside of `bee`, and then actually renaming it to just `json` when ported out, and using a named import when adding back the dependency.

Reply:
Cool, thanks @mortelli. 👍